lib/mousestats/index.js

```diff
@@ -31,7 +31,7 @@ var MouseStats = module.exports = integration('MouseStats')
 MouseStats.prototype.initialize = function(page){
   var number = this.options.accountNumber;
   var path = number.slice(0,1) + '/' + number.slice(1,2) + '/' + number;
-  var cache = Math.floor(new Date().getTime() / 60000);
+  var cache = Math.floor(new Date().getTime() / 180000);
```

Comment:
Curious about the coming up with the number 18000 here, could you let me know why? For context, docs you've linked to use 600000.